test/python/test_with_schedules.py

```diff
@@ -474,6 +474,28 @@ def ppsp_verified_test(self, input_file_name, use_separate_algo_file=False):
                 break
         self.assertEqual(test_flag, True)
 
+    def tc_verified_test(self, input_file_name, use_separate_algo_file=False):
+        if use_separate_algo_file:
+            self.basic_compile_test_with_separate_algo_schedule_files("tc.gt", input_file_name)
```

Comment:
Can you also add a cleaned up version of "tc.gt" to the apps directory?

Reply:
Yep!